dynalab/tasks/common.py

```diff
@@ -48,6 +48,10 @@ def _get_mock_context(model_name):
             print("Getting model response ...")
             response = handle_func(mock_data, mock_context)
             print(f"Your model response is {response}")
+            try:
+                json.dumps(response)
+            except Exception as e:
+                raise Exception("The model response isn't serializable to json !") from e
```

Comment:
This allows catching non serializable objects in the response returned by the user.

Reply:
Please raise RuntimeError :)

Reply:
done.